bp-web-client/angular-app/src/app/components/app-contact/app-contact.component.ts

```diff
@@ -80,14 +80,21 @@ export class ContactComponent implements OnInit {
   budgets: any = ['Slide for budget amount'];
   timelines: any = ['Slide for time frame'];
   timeframe: any = [0, 35];
-  years: any = ['1980 and older', 1981, 1982, 1983, 1984, 1985, 1986, 1987, 1988, 1989, 1990, 1991, 1992, 1993, 1994, 1995, 1996, 1997, 1998, 1999, 2000, 2001, 2002, 2003, 2004, 2005, 2006, 2007, 2008, 2009, 2010, 2011, 2012, 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020];
+  years = [];
 
   ngOnInit() {
-
+    const yearList = new Date().getFullYear();
```

Comment:
Nice job with logic!  However, please put all this logic in a function so that it is reusable. 
i.e. `this.years = this.calculateYears(new Date().getFullYear(), 40);`

Reply:
Thank you! I resolved the issue as well as the conflicts.